index.html

```diff
@@ -90,12 +140,21 @@ <h2 class="visually-hidden">Phone control</h2>
                 <!-- <img src="../images/phone.png" alt="" class="phone-border" /> -->
 
                 <div class="phone-status-bar">
-                    <p>12:34</p>
+                   
+                    <div id="time" class="time"></div>
                     <div class="phoneIcon">
+
+
                         
                         <img src="./images/networkIcon.svg" alt="" />
                         <img src="./images/wifiIcon.svg" alt="" />
-                        <img src="./images/batteryIcon.svg" alt="" />
+                        
+                          <div id="battery" class="battery">
+                            <div id="battery-level" class="battery-level"></div>
+                            <div id="level" class="level">(battery level unknown)</div>
```

Comment:
Misschien niet zo'n lange tekst als het niet ondersteund wordt. Dit past niet in de ruimte van het batterij-icoontje. Je kan hem standaard op 50% zetten denk ik.

Reply:
fixed